packages/library/cozy-lib/templates/_render_custom_forms_prefill.tpl

```diff
@@ -0,0 +1,16 @@
+{{- define "renderValues" -}}
```

Comment:
```suggestion
{{- define "cozy-lib.frontend.renderValues" -}}
```
Namespace the custom templates by chart name (cozy-lib) and function (e.g. frontend).